### ISSUE TITLE
Support import via 'git = ...'

### ISSUE THREAD

**Issue by cameronelliott** [ANLAB-KAIST/rust-dpdk#62]
The README says:

```
Now add rust-dpdk to your project's Cargo.toml and use it!

[dependencies]
rust-dpdk = { git = "https://github.com/ANLAB-KAIST/rust-dpdk.git" }
```

I think this is impossible, because /Cargo.toml name `dpdk-sys` does not match name in /dpdk-sys/Cargo.toml `rust-dpdk-sys`

I was not able to import due to the name mismatch.
I submitted PR in #61

**Comment by leeopop** [comment]
Hi, it seems the documentation is not updated yet.

I expect to use this package as following cargo dep.
```
rust-dpdk-sys = { git = "https://github.com/ANLAB-KAIST/rust-dpdk", branch = "main" }
```

I referenced the directory structure from `https://github.com/tokio-rs/tokio`

Would it be enough to change the README file? 
Currently, this library is extreamly unstable so we have no plan to publish it via crate.io yet.

**Comment by leeopop** [comment]
Hi, it appears that the documentation hasn't been updated yet.

I referred to the directory structure from https://github.com/tokio-rs/tokio and plan to use this package as follows in the Cargo dependency:

```toml
rust-dpdk-sys = { git = "https://github.com/ANLAB-KAIST/rust-dpdk", branch = "main" }
````

I skimmed through your PR and noticed it seems to serve two purposes:
1) Fixing the dependency issue.
2) Preparing for publication on crates.io.

Currently, this library is extremely unstable, so we have no plans to publish it on crates.io just yet.
Would updating the README file suffice for addressing the first issue?
